ui/modal/info_modal.go

```diff
@@ -259,6 +260,13 @@ func (in *InfoModal) SetupWithTemplate(template string) *InfoModal {
 		customTemplate = totalValueInfo(in.Theme)
 	case BondStrengthInfoTemplate:
 		customTemplate = bondStrengthInfo(in.Theme)
+	default:
+		prepTemplate := func(th *cryptomaterial.Theme) func(gtx C) D {
+			return func(gtx layout.Context) layout.Dimensions {
+				return th.Body2(template).Layout(gtx)
+			}
+		}
+		customTemplate = append(customTemplate, prepTemplate(in.Theme))
```

Comment:
What's going on here?

Reply:
Modals use the custom templates to display content. I’ve just added a default handling for translated text which can be provided is instead of a template.

Reply:
So it assumes that the provided template is a text to be displayed? Sounds buggy.

Reply:
> So it assumes that the provided template is a text to be displayed?

Yes. This is a legacy method used to display text, and we could use a default case in this scenario.